README.md

```diff
@@ -1 +1,5 @@
-# skyux-tree-view
\ No newline at end of file
+# @skyux/tree-view
```

Comment:
I know it's gross, but we should probably stick to `@skyux/angular-tree-component` so that the third-party library's NPM package name is apparent. We can still use the `<sky-tree-root-wrapper>` name, however.

Reply:
I'll rename the repo, too.